hub/core/serialize.py

```diff
@@ -143,7 +143,7 @@ def get_header_from_url(url: str):
     enc_dtype = np.dtype(hub.constants.ENCODING_DTYPE)
     itemsize = enc_dtype.itemsize
 
-    headers = {"Range": "bytes=0-100"}
+    headers = {"Range": "bytes=0-99"}
```

Comment:
Didn't get why we need this change. Can you pls clarify it a bit more?

Reply:
Wanted to get just 100 bytes but the range header includes both lower and upper bound so it used to be 101 bytes. This change is not actually necessary.